packages/terra-clinical-slide-group/docs/README.md

```diff
@@ -0,0 +1,27 @@
+# Terra Clinical Slide Group
+
+The SlideGroup is a Component that utilizes the `react-transition-group` library to present a stack of components in an
+animated fashion.
+
+The last component in the group is visible, and all others are hidden. While the hidden components are not visible and
+marked with the `aria-hidden` attribute, they are not actually unmounted.
```

Comment:
No more so than if someone presented a component modally that was already present on the main page. Ids should generally be avoided by folks producing embeddable components, as they cannot guarantee there won't be duplication of those components on the same page.

We did talk about the performance implications of keeping all these mounted. I think that's something we should look into in the future.